server/e2e/jobs/specs/library.e2e-spec.ts

```diff
@@ -50,20 +50,96 @@ describe(`${LibraryController.name} (e2e)`, () => {
       const onlineAssets = await api.assetApi.getAllAssets(server, admin.accessToken);
       expect(onlineAssets.length).toBeGreaterThan(1);
 
-      await restoreTempFolder();
+      await fs.promises.rm(`${IMMICH_TEST_ASSET_TEMP_PATH}/albums/nature/silver_fir.jpg`);
 
-      await api.libraryApi.scanLibrary(server, admin.accessToken, library.id);
+      await api.libraryApi.scanLibrary(server, admin.accessToken, library.id, { checkForOffline: true });
 
       const assets = await api.assetApi.getAllAssets(server, admin.accessToken);
 
       expect(assets).toEqual(
         expect.arrayContaining([
           expect.objectContaining({
             isOffline: true,
-            originalFileName: 'el_torcal_rocks',
+            originalFileName: 'silver_fir',
           }),
           expect.objectContaining({
-            isOffline: true,
+            isOffline: false,
+            originalFileName: 'tanners_ridge',
+          }),
+        ]),
+      );
+    });
+
+    it('should not offline a missing file when performing regular scan', async () => {
```

Comment:
I'm not sure it makes sense to have a test for this. Is it undesirable for it to offline a missing file?

Reply:
Thanks for spotting